vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -3,14 +3,28 @@
 
 namespace Microsoft.ReactNative.Bridge
 {
+	[webhosthidden]
+	enum ViewManagerPropertyType
+	{
+		Boolean,
+		Number,
+		String,
+		Array,
+    Color,
+	};
+
 	[webhosthidden]
 	interface IViewManager
```

Comment:
>IViewManager [](start = 11, length = 12)

Can we do it similar to INativeModuleBuilder instead to avoid usage of the IMapView that forces us to do the value boxing/unboxing?